tutor/src/screens/assignments/builder.js

```diff
@@ -21,7 +20,6 @@ const Header = styled.div`
 
 const HeaderStep = styled.div`
   font-size: 1.2rem;
-  text-transform: uppercase;
```

Comment:
I generally like to keep this in CSS to help screenreaders not mistake it as an acronym, but testing this one with NVDA, it seems to be alright.